programs/dasheri/src/pool/instructions/create_mango_account.rs

```diff
@@ -6,9 +6,15 @@ use solana_program::program::invoke_signed;
 #[derive(Accounts)]
 #[instruction(account_num: u64, bump: u8)]
 pub struct PoolCreateMangoAccount<'info> {
+    // REVIEW: So this can be done for any mango program/group permissionlessly?
```

Comment:
one can definitely hard code the address, I will add a todo

Reply:
do you think any other check then target address would be useful?

Reply:
no, I expect it is fine to be able to use this program with any mango program/group; definitely fine for the example